pkgs/applications/blockchains/snarkvm/default.nix

```
@@ -0,0 +1,26 @@
+{ fetchFromGitHub, lib, rustPlatform, pkg-config, openssl }:
+rustPlatform.buildRustPackage rec {
+  pname = "snarkVM";
+  version = "1.1.0";
+
+  src = fetchFromGitHub {
+    owner = "ProvableHQ";
+    repo = pname;
+    rev = "v{$version}";
+    sha256 = "sha256-U0ni6KulKl+DLUgWGl7Qy5NHahuu3wrmPxbnJApuchw=";
+  };
+
+  cargoHash = "";
+
+  nativeBuildInputs = [ pkg-config ];
+
+  buildInputs = [ openssl ];
+
+  meta = with lib; {
+    description = "zero-knowledge virtual machine (ZK-VM)";
+    homepage = "https://github.com/ProvableHQ/snarkVM";
+    maintainers = [ ];
+    license = [ licenses.asl20 ];
+    platforms = platforms.unix;
+  };
```

```suggestion
  meta = with lib; {
    description = "Zero-knowledge virtual machine (ZK-VM)";
    homepage = "https://github.com/ProvableHQ/snarkVM";
    maintainers = [ ];
    license = [ lib.licenses.asl20 ];
    platforms = lib.platforms.unix;
  };
```

Also, please add yourself to the maintainers list